doc/developers-guide/mongoose_wpool.md

```diff
@@ -0,0 +1,27 @@
+# mongoose_wpool
+
+All the [outgoing pools] configured by the `outgoing_pools` option are hidden behind `mongoose_wpool` API.
```

Comment:
All the [outgoing pools] configured by the `outgoing_pools` option are hidden behind the `mongoose_wpool` API.